onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -278,17 +312,21 @@ def run_tensorflow(use_gpu, model_names, model_class, precision, batch_sizes, se
                 if max_input_size is not None and sequence_length > max_input_size:
                     continue
 
-                logger.info("Run Tensorflow on {} with input shape {}".format(model_name, [batch_size, sequence_length]))
+                logger.info("Run Tensorflow on {} with input shape {}".format(model_name,
+                                                                              [batch_size, sequence_length]))
 
                 import random
                 rng = random.Random()
                 values = [rng.randint(0, config.vocab_size - 1) for i in range(batch_size * sequence_length)]
                 input_ids = tf.constant(values, shape=(batch_size, sequence_length), dtype=tf.int32)
 
                 try:
+                    # Disable both for better inference perf
```

Comment:
I heard that xla help performance. Do you know why they are not in your test.

Reply:
I am following the huggingface benchmark that defaults use_xla to false.
I also run tests with use_xla and find that:
cpu: ~1.1x slower
gpu: ~1.2x slower